### DISCUSSION TITLE
Get "Pro" tag in Highlights

### DISCUSSION THREAD

**Discussion by runesatsdev** [community/community#49495] [Codespaces] [answered]
### Select Topic Area

Question

### Body

I saw that some guys have "pro" tag in the Highlights.
How can I get that tag?

**Comment by microgift** [comment, marked as answer]
To get a Pro tag on your GitHub profile, you need to follow the below steps:

Go to the GitHub website and sign in to your account.

Click on your profile picture icon located in the top right corner.

From the dropdown menu, select "Settings."

In the left sidebar menu, click on the option "Billing & plans."

Under the "Plan" section, you will see an option for "Upgrade to Pro." Click on it.

You will be taken to a page where you can choose the plan that meets your requirement. Select the GitHub Pro plan.

Enter your payment information, and then click on the "Review your order" button.

Finally, click on the "Complete order" button to complete the purchase process.

Once the purchase is complete, you will be able to access all the features of GitHub Pro, including the Pro tag in highlights.

**Comment by W-N-R** [comment]
good

**Comment by Mobeen050** [comment]
How i got  pro on my University Account with Student Pack ?
